root/edit/details/ChangeWikiDoc.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * @flow strict
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import DBDefs from '../../static/scripts/common/DBDefs';
+
+type ChangeWikiDocEditT = {
+  ...EditT,
+  +display_data: {
+    +new_version: number | null,
+    +old_version: number | null,
+    +page: string,
+  },
+};
+
+type Props = {
+  +edit: ChangeWikiDocEditT,
+};
+
+const ChangeWikiDoc = ({
+  edit,
+}: Props): React.Element<'table'> => {
+  const display = edit.display_data;
+  const page = display.page;
+  const oldVersion = display.old_version;
+  const newVersion = display.new_version;
+  const baseLink = `//${DBDefs.WIKITRANS_SERVER}/index.php?title=${page}`;
+  const oldLink = oldVersion == null
+    ? null
+    : baseLink + `&oldid=${oldVersion}`;
+  const newLink = newVersion == null
+    ? null
+    : baseLink + `&oldid=${newVersion}`;
+  const diffLink = (oldVersion == null || newVersion == null)
+    ? null
+    : baseLink + `&diff=${newVersion}&oldid=%${oldVersion}`;
```

Comment:
there's a seemingly-extraneous % here
```suggestion
    : baseLink + `&diff=${newVersion}&oldid=${oldVersion}`;
```